llmware/library.py

```diff
@@ -393,15 +507,44 @@ def set_knowledge_graph_status (self, status_message):
     def get_and_increment_doc_id(self):
 
         """ Convenience method in library class - mirrors method in LibraryCatalog - increments, tracks and provides a
-        unique doc id for the library """
+        unique doc id for the library 
+        
+            Returns
+            -------
+            unique_doc_id : int
+                The new unique document ID for the library.
+        """
 
         unique_doc_id = LibraryCatalog(self).get_and_increment_doc_id(self.library_name)
         return unique_doc_id
 
     def set_incremental_docs_blocks_images(self, added_docs=0, added_blocks=0, added_images=0, added_pages=0,
                                            added_tables=0):
 
-        """ Updates the library card with incremental counters after completing a parsing job """
+        """ Updates the library card with incremental counters after completing a parsing job 
```

Comment:
Delete space before you start the doc and add full stop.
```suggestion
        """Updates the library card with incremental counters after completing a parsing job.
```